### ISSUE TITLE
Installation of packages with pip

### ISSUE THREAD

**Issue by p-j-smith** [carpentries-incubator/python-intermediate-development#104]
Hi, this seems like a fantastic course on Python!

I've been reading through the first lesson and just have a minor point. The lessons suggests to use pip to install packages as follows:
```
(venv) $ pip3 install numpy
```
However, [the recommended way](https://pip.pypa.io/en/stable/user_guide/#running-pip) is:
```
(venv) $ python3 -m pip install numpy
```
This is to ensure that the correct pip is being used (i.e. the one that corresponds to the activated python3 interpreter).

**Comment by smason** [comment]
Interesting! I personally tend to use `pip install foo` (now Python 2 is gone) and see even the (official) linked docs seem to have a mixture of usages.

That said, core Python developer Brett Cannon suggests using `python -m pip`, and has a more detailed explanation in https://snarky.ca/why-you-should-use-python-m-pip/ of when it's actually required, and might be a useful pointer to reference

I should get into the habit of doing this!  I fit into the "always using an activated environment" group, but Brett's article above was an interesting read for when these would do different things.

I could turn this into a pull request if that would help

**Comment by anenadic** [comment]
Feels like this is more of a convention, `python3 -m pip install numpy` may be "more correct" and certainly more consistent with the rest of material. Perhaps still not prevalent in use - is this down to how the tools are used conventionally? Will add a note on the alternative command `python3 -m pip install numpy` but keep the old command as it seems more prevalent among developers.

**Comment by anenadic** [comment]
Fixed via #117 .
